### PR TITLE
Rev gatk public to 4.alpha.2-260-ge8d00ca-20170508.215840-1

### PR DESCRIPTION
**Pull request by droazen**
-Updates to GenomicsDBImport (add --batchSize and --consolidate,
 and provide a way to avoid downloading all headers up-front)

-Fix to RMSMappingQuality (don't blow up when 0 reads)

-Close threads properly in NIO code

-Allow HomRefBlock to handle GQs > 99 without blowing up.

### CODE REVIEW

**Comment by droazen** [comment]
@lbergelson please review

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java [hunk 1]
```diff
@@ -50,10 +50,10 @@
      *
      * This argument allows you to set the GQ boundaries. HC expects a list of multiple GQ threshold values. To pass
      * multiple values, you provide them one by one with the argument, as in `-GQB 10 -GQB 20 -GQB 30` and so on. Note
-     * that GQ values are capped at 99 in the GATK.
+     * that GQ values are capped at 99 in the GATK, so values must be integers in the range [1, 99].
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java]: I think the max may be 100 if you want to capture 99's in a band with anything else.

**Comment by lbergelson** [comment]
Thumbs up from me. We can address that comment later since we need this now.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java (hunk 1 above)
Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java]: These specify the lower inclusive bounds, so I think 99 is correct (perhaps the old documentation should be updated if this is not clear).  I believe that putting in 100 causes an exception to be thrown, but I'm on my phone so I can't easily check this.

Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java]: Back at a computer---actually, it looks like I updated the documentation and changed the meaning of the GQB arguments in https://github.com/broadinstitute/gsa-unstable/pull/1451, along with changing the necessary tests.  These changes were not ported over in the GATK4 public/protected PRs by @ronlevine. so we should issue an additional PR to add these changes.

Comment (samuelklee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java]: Filed https://github.com/broadinstitute/gatk-protected/issues/1029.

**Pull request merged**